### DISCUSSION TITLE
Chapter Expansion Plan

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#313] [General]
Dear Albert, some of the chapters state its incompletment and plan to expand it.
How about the schedule? 😄

**Comment by ialbert** [comment]
the new books are typically where those expansions occur, sometimes via chapters, sections, and so on. 

However, it is true that I should revisit the mentions of these plans and either point the reader to the new location or delete those sections if a sufficiently long time has passed and no developments have been made. Alas I don't have a list of pointers, perhaps you can compile this list and send it to me (private email is also fine) and I'll see what is going on with each.

**Reply by TullyMonster** [reply]
Well, I think when I finish reading the book, I'll make this list.
